doc/releases/release-notes-3.6.rst

```diff
@@ -261,6 +263,33 @@ Drivers and Sensors
   * The Legacy Virtual Register defines have been renamed from ``I3C_DCR_I2C_*``
     to ``I3C_LVR_I2C_*``.
 
+  * Added the ability to specify a start address when searching for a free I3C
+    address to be reserved. This requires a new function argument to
+    :c:func:`i3c_addr_slots_next_free_find`.
+
+  * Added a field named ``num_xfer`` in :c:struct:`i3c_msg` and
+    :c:struct:`i3c_ccc_taget_payload` as an output to indicate the actual
+    number of bytes transferred.
+
+  * ``i3c_cdns``:
```

Comment:
You can use ":dtcompatible:`...`" for referring to device drivers through their bindings.

Reply:
My intention is to have something that corresponds to the file name of the drivers (for easier reference). I have amended to actually spell out the names of the driver, and the path to the driver source files.